opentracing-spring-web/src/main/java/io/opentracing/contrib/spring/web/interceptor/HandlerInterceptorSpanDecorator.java

```diff
@@ -71,6 +71,7 @@ public void onPreHandle(HttpServletRequest httpServletRequest, Object handler, B
             metaData = HandlerUtils.methodName(handler);
             if (metaData != null) {
                 logs.put(HandlerUtils.HANDLER_METHOD_NAME, metaData);
+                span.setOperationName(metaData);
```

Comment:
Pushed change for the previous comment - but wasn't sure what you meant here? Do you mean log an error if the `metaData` is null?